changelog/fix-4847-saving-settings-when-enabled-pm-setting-faulty

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Repair a potentially corrupted state of enabled payment method settings, that causes an error while saving settings. Example - A frequently reported issue of `Link` payment method appearing in enabled payment methods array of non-us clients leading to error while trying to save.
```

Comment:
We should make this entry shorter. I'd kee the first sentence only,

Reply:
Done! Cropped to just the first sentence. . Marking as resolved.